admin/atlas_client.go

```diff
@@ -1,5 +1,5 @@
 package admin // import "go.mongodb.org/atlas-sdk/admin"
-// Autogenerated based on OpenAPI Version: 2.0~1056ca6d98
+// Autogenerated based on OpenAPI Version: 2.0~e40c18c004
```

Comment:
@gssbzn IMHO we should have a backend filter that removes that element from the schema as it does spam us in long run. Will create jira.